recipes/jinja2cpp/all/conanfile.py

```diff
@@ -42,11 +42,15 @@ def configure(self):
     def requirements(self):
         self.requires("boost/1.76.0")
         self.requires("expected-lite/0.5.0")
-        self.requires("fmt/6.2.1") # not compatible with fmt >= 7.0.0
         self.requires("optional-lite/3.4.0")
         self.requires("rapidjson/cci.20200410")
         self.requires("string-view-lite/1.6.0")
         self.requires("variant-lite/2.0.0")
+        if self.version == "1.1.0":
+            self.requires("fmt/6.2.1") # not compatible with fmt >= 7.0.0
+        if self.version == "1.2.1":
```

Comment:
I would use an `else` here.
Otherwise adding a version will break the build.
```suggestion
        else:
```

Reply:
Right, thanks!